RELEASING/README.md

```diff
@@ -211,6 +203,14 @@ Then tag the final release:
 Now you can announce the release on the mailing list, make sure to use the
 proper template
 
+### Publishing a Convenience Release to PyPI
+From the root of the repo running ./pypi_push.sh will build the
+Javascript bundle and echo the twine command allowing you to publish
+to PyPI. You may need to ask a fellow committer to grant
+you access to it if you don't have access already. Make sure to create
+an account first if you don't have one, and reference your username
+while requesting access to push packages.
+
```

Comment:
Unrelated, but the `RELEASING` directory might be a more appropriate location for the `pypy_push.sh` script.

Reply:
right or even `scripts/`, though afaic feel free to merge as is and do a follow up PR or issue

Reply:
Agree, the release docs and scripts are improving, still would like to fully (or almost) automating this process